src/Routes.js

```diff
@@ -11,6 +13,8 @@ const Routes = () => {
       <Switch>
         <Route exact path="/" component={Main} />
         <Route exact path="/upload" component={Upload} />
+        <Route exact path="/footer" component={Footer} />
+        <Route exact path="/login" component={Login} />
       </Switch>
```

Comment:
Footer 컴포넌트는 경로를 타고 들어가서 보여져야 하는 컴포넌트가 아니기 때문에 별도 경로를 주시기 보다는 아래와 같이 Switch 바깥으로 빼주는 편이 맞아보입니다.
```suggestion
      <Switch>
        <Route exact path="/" component={Main} />
        <Route exact path="/upload" component={Upload} />
        <Route exact path="/login" component={Login} />
      </Switch>
      <Footer />
```